### PR TITLE
exporting ClientInfo functions for better reusability

### PR DESCRIPTION
**Pull request by lalimasharda**

### CODE REVIEW

File: change/@azure-msal-common-53e36e4a-c298-4b42-aada-9dd3ef0c7deb.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "refactoring code for reusability in msal 1p #6886",
```
Comment (tnorling) [inline change/@azure-msal-common-53e36e4a-c298-4b42-aada-9dd3ef0c7deb.json]:
```suggestion
  "type": "minor",
  "comment": "Export ClientInfo APIs #6886",
```

File: lib/msal-common/src/index.ts [hunk 2]
```diff
@@ -33,7 +35,8 @@ export { AuthorityType } from "./authority/AuthorityType";
 export { ProtocolMode } from "./authority/ProtocolMode";
 // Cache
 export { CacheManager, DefaultStorageClass } from "./cache/CacheManager";
-export { AccountCache, AccessTokenCache, IdTokenCache, RefreshTokenCache, AppMetadataCache, ValidCacheType, ValidCredentialType } from "./cache/utils/CacheTypes";
+export { AccountCache, AccessTokenCache, IdTokenCache, RefreshTokenCache, AppMetadataCache, ValidCacheType, 
+    ValidCredentialType } from "./cache/utils/CacheTypes";
```
Comment (tnorling) [inline lib/msal-common/src/index.ts]: Why did some of these get pushed onto new lines?

Comment (lalimasharda) [reply lib/msal-common/src/index.ts]: Just to improve readability so that you don't have to scroll a lot.

Comment (tnorling) [reply lib/msal-common/src/index.ts]: I see. Can you set your IDE to wrap? Given everyone has different screen sizes this feels a bit arbitrary.

Comment (lalimasharda) [reply lib/msal-common/src/index.ts]: Fixed

**Review by peterzenz — APPROVED** [review]

**Review by tnorling — APPROVED** [review]

**Pull request merged**